content/games/factions/locations.md

```diff
@@ -0,0 +1,464 @@
+
+The side that rolled higher places the first Fissure token (3" across). The sides then
+alternate placing the remaining Fissure tokens. Fissures are impassable terrain and
+everything within 6" of them is difficult terrain. After the Fissure tokens have been placed,
+place an equal number of Soothing Stone tokens (1" across) by the same process.
+
+### Fissures
+
+After both sides have taken their turn, roll 1d6 for each Fissure.
+
+| Result | Effect                                                                                   |
+| :----: | :--------------------------------------------------------------------------------------- |
+|   1    | Spawn a Magmatic Hound by the well.                                                      |
+|  2-3   | The well belches flame and magma. Roll 6d6 against every Group within 6", hitting on 3+. |
+|   4+   | The well burbles and shakes, steaming ominously.                                         |
+
+### Soothing Stones
+
+Any Group may attempt a Move activation to pick up a Soothing Stone. Groups carrying a Soothing
+Stone are ignored by Magmatic Hounds. A Group adjacent to a Fissure and carrying a Soothing Stone
+may freely activate to throw it into the Fissure, calming it. If they do so, remove the Fissure
+token and reduce the clock towards Pyroclasm by 1. For each calmed Fissure, +2 VP if the Pyroclasm
+is avoided.
+
+For every Group carrying a Soothing Stone when the Pyroclasm starts, roll a d6. If the result is
+equal to or less than their T, they survive.
+
+### Magmatic Hounds
+
+After rolling for every Fissure, every Magmatic Hound must activate. If any of them fail, the
+rest continue to activate. If a Magmatic Hound cannot Attack a Group, it must move towards one if
+they are within 12". Otherwise, it Howls.
+
+|     ME      |  MI   |    MO    |  FS   |   R   |   T   | Traits                          |
+| :---------: | :---: | :------: | :---: | :---: | :---: | :------------------------------ |
+| 5+ / 3+ / 6 |   -   | 6+ / 10" |   6   |  3+   |   4   | Defiant, Howl, Nimble, Reckless |
+
+Defiant
+: If a Magmatic Hound is not Shaken and not already in melee, when anyone successfully activates to
+  Attack them, they may test at 7+ to meet the enemy halfway and count both sides as attacking for
+  to-hit rolls.
+
+Howl
+: Activate on a 5+. If the result is less than 10, the Magmatic Hound summons another Magmatic
+  Hound which does not activate this turn. Otherwise, advance the clock towards Pyroclasm by 1.
+
+Nimble
+: Magmatic Hounds treat difficult terrain as normal terrain instead.
+
+Reckless
+: If a Magmatic Hound is not Shaken, it must activate to Attack if possible.
+
+### Pyroclasm
+
+After the clock toward Pyroclasm is full, it arrives in a fury of tephra. Any Group not carrying a
+Soothing Stone is immediately Routed and the skirmish ends. If the clock toward Pyroclasm is reduced
+to less than zero, the Pyroclasm is averted.
```

Comment:
wait okay. so i can't quite work out how the pyroclasm clock works just from what's in the setup. they roll the 2d6 etc, the higher result is how many rounds until pyroclasm. that's a countdown. so i'd like to structure it such that you tick the number _down_ instead of up, as the pyroclasm approaches. 

SO we just need to define an upper bound where the volcano goes dormant (instead of 0, as here). if that makes sense.

Reply:
Also, if the pyroclasm is averted, do we end the skirmish? Or continue to play last-man-standing style? best to specify

Reply:
Good shout, I was struggling with this myself when drafting. Rereading the draft, the procedure _as written_ is:

1. Each side rolls 1d6.
2. Higher result is the turns to Pyroclasm, lower result is the number of Fissures and Soothing Stones to place.
3. After ?? turns (not clear: turns _per player_ or turns by both players - I didn't explicitly state when the clock advances), the Pyroclasm arrives.
4. Players can order any of their Groups carrying a Soothing Stone who are adjacent to a Fissure to throw it in, reducing the Pyroclasm clock by 1.
5. If the Pyroclasm clock is reduced to _less_ than zero, the Pyroclasm is averted.

As you noted, one problem with this is counting up instead of down. The text should also explicitly state when the clock advances.

Thinking about the rewrite, there may be scenarios with no possibility of averting the pyroclasm (if the lower roll is a 1, it's functionally impossible, I think).

Instead, maybe:

1. Each side rolls 1d6. The total is the number of turns until the Pyroclasm. The lower result is the number of Fissures, the higher is the number of Soothing Stones.
2. The Pyroclasm clock ticks down after both sides have taken a turn.
3. Throwing a Soothing Stone into a Fissure adds 1d6 ticks to the Pyroclasm clock.
4. If the Pyroclasm clock ever exceeds 15, it is averted and play continues depending on the Scenario's guidelines.

That should mean there are more stones but fewer fissures and every thrown stone gives a _pretty good_ chance at helping to avert the Pyroclasm without making it a done-deal.

Reply:
That revised procedure seems good to me. I haven't run the math, but is 1d3 better than 1d6 for Soothing Stone effect? I suspect 1d6 is better, because then the whole skirmish kind of revolves around doing it / preventing it, depending on which side you are. Very uhhh soccer.

Reply:
I've been trying to avoid having any dice other than a d6 if I can help it. I'll incorporate this procedure for now, I really want to play some of these locations in the near future.